src/Plugins/DBFTPlugin/Consensus/ConsensusService.Check.cs

```diff
@@ -46,8 +46,8 @@ private bool CheckPrepareResponse()
                 }
 
                 // Timeout extension due to prepare response sent
-                // around 2*15/M=30.0/5 ~ 40% block time (for M=5)
-                ExtendTimerByFactor(2);
+            // around 4*3/M=12.0/5 ~ 80% block time (for M=5)
+            ExtendTimerByFactor(4);
```

Comment:
This is a tricky one. In our tests we have not changed any of these parameters and they look scalable by their nature, if we use lower block times we expect lower waiting times in general as well. So I'd not change them unless we have some real proven need to.

Reply:
This is needed, I have many tests on that for NeoCompiler.
I had also run in the past with server spread along globe.
I suggest this parameters.
But you guys do not want to change we can try with original. But this is my recommended for now.

Reply:
@roman-khimov, you are right that this method is really tricky....I remember exactly why we created that....And that is really tricky...aheuaheua Luckly we had this idea during creating of dBFT 2.0, otherwise (other changes would be necessary)......